src/test/java/au/com/gritmed/pay/PaymentNotificationsConsumerTest.java

```diff
@@ -50,13 +55,35 @@ public void consumingPaymentNotifications() throws Exception {
                 if (++counter % 10000 == 0) {
                     System.out.println(LocalDateTime.now() + ": Consumed " + counter + " payments");
                 }
+                if (counter % 100000 == 0) {
+                    showPause(lastTime);
+                    lastTime = System.currentTimeMillis();
+                }
             }
         } while (true);
 
     }
 
+    private void showPause(long lastTime) {
```

Comment:
this is here just to give you an idea of how many times the pauser is invoked - how many times/how long we have to wait for the payments queue